src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -612,18 +601,23 @@ public AdsBaseObject triggerAdSlot(String slotName) {
     return this;
   }
 
-  public void verifyExpandedAdVisibleInSlot(String slotSelector, WebElement slot) {
-    waitForSlotExpanded(slot);
+  private void triggerBLB() {
+    scrollToFooter();
 
-    boolean adVisible = new AdsComparison().isAdVisible(slot, slotSelector, driver);
+    simulateUserActivity(Duration.ofSeconds(3));
+  }
 
-    extractGptInfo(slotSelector);
+  private void simulateUserActivity(Duration duration) {
+    simulateUserActivity(duration, Duration.ofMillis(500));
+  }
 
-    if (!adVisible) {
-      throw new WebDriverException("Ad is not present in " + slotSelector);
+  private void simulateUserActivity(Duration duration, Duration waitDuration) {
+    while (!duration.isNegative()) {
+      jsActions.scrollBy(0, 100);
+      wait.forX(waitDuration);
+      jsActions.scrollBy(0, -100);
+      duration = duration.minus(waitDuration);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "duration". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)